### PR TITLE
upload script for uninstall cloud native monitoring resources

### PR DESCRIPTION
**Pull request by niuxiaojuan-github**

### CODE REVIEW

**Review by wlijiancp4mcm — APPROVED** [review]

**Comment by stacyped** [comment]
Can you also update the CHANGELOG.md? Thank you

**Comment by niuxiaojuan-github** [comment]
> Can you also update the CHANGELOG.md? Thank you

@stacyped I have changed CHANGELOG.md, and moved cnmon/uninstall_cnmon.sh into scripts folder

File: scripts/cnmon/uninstall_cnmon.sh [hunk 1]
```diff
@@ -0,0 +1,218 @@
+#!/bin/sh
+
+namespace=cp4mcm-cloud-native-monitoring
+if [ ! -z $1 ]
+then
+  namespace=$1
+fi
+echo "Uninstall Cloud Native Monitoring from namespace $namespace..."
+
+echo "Please make sure you have changed the label to ibm.com/cloud-native-monitoring: \"disabled\" at IBM Cloud Pak® for Multicloud Management hub cluster for this managed cluster"
```
Comment (stacyped) [inline scripts/cnmon/uninstall_cnmon.sh:10]: Can this be "Ensure that you have changed the label to ibm.com/cloud-native-monitoring: \"disabled\" on the IBM Cloud Pak® for Multicloud Management hub cluster for this managed cluster.

File: scripts/cnmon/uninstall_cnmon.sh [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  y*|Y*) ;;
+
+  *) exit 0
+  ;;
+esac
+
+kubectl cluster-info
+if [ ! $? -eq 0 ]
+then
+  echo "Please login the cluster and rerun the script"
```
Comment (stacyped) [inline scripts/cnmon/uninstall_cnmon.sh:24]: "Log in to the cluster and rerun the script."

File: scripts/cnmon/uninstall_cnmon.sh [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+echo ""
+if [ ! $nscnt -eq 0 ]
+then
+  echo "Deleting namespace $namespace..."
+  echo "Running pods in namespace $namespace"
+  kubectl get pods -n $namespace
+  echo ""
+
+  echo "If you have deployments other than Cloud Native Monitoring installed in namespace $namespace, please uninstall all the other deployments before run this script to delete namespace."
```
Comment (stacyped) [inline scripts/cnmon/uninstall_cnmon.sh:181]: "If you have deployments other than Cloud Native Monitoring installed in namespace $namespace, you must uninstall all of the other deployments before you run this script to delete this namespace."
  echo "***** NOTE THAT ALL RESOURCES IN NAMESPACE $namespace WILL BE DELETED!!! *****"

File: scripts/cnmon/uninstall_cnmon.sh [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    echo "Waiting 10 seconds..."
+    sleep 10
+    podcnt=$(kubectl get pods -n $namespace --no-headers=true --ignore-not-found=true | wc -l)
+    i=$((i+1))
+  done
+
+  if [ $podcnt -gt 0 ]
+  then
+    kubectl get pods -n $namespace
+    echo "There are running pods, you might have other deployments installed in namespace $namespace, please check and uninstall all deployments before rerun this script to delete namespace $namespace."
```
Comment (stacyped) [inline scripts/cnmon/uninstall_cnmon.sh:206]: Maybe something like "There are running pods. If you have other deployments installed in namespace $namespace, check and uninstall all deployments before you rerun this script to delete namespace $namespace."

File: scripts/cnmon/uninstall_cnmon.sh (hunk 1 above)
Comment (niuxiaojuan-github) [reply scripts/cnmon/uninstall_cnmon.sh]: fixed

File: scripts/cnmon/uninstall_cnmon.sh (hunk 2 above)
Comment (niuxiaojuan-github) [reply scripts/cnmon/uninstall_cnmon.sh]: fixed

File: scripts/cnmon/uninstall_cnmon.sh (hunk 3 above)
Comment (niuxiaojuan-github) [reply scripts/cnmon/uninstall_cnmon.sh]: fixed

File: scripts/cnmon/uninstall_cnmon.sh (hunk 4 above)
Comment (niuxiaojuan-github) [reply scripts/cnmon/uninstall_cnmon.sh]: fixed

**Comment by niuxiaojuan-github** [comment]
@stacyped thank your for reviewing, I have fixed the prompt messages.

**Pull request merged**